server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -225,6 +231,9 @@ async fn list_messages(
 pub struct CreateMessageQueryParams {
     #[serde(default = "default_true")]
     with_content: bool,
+    /// If set to true a 200 OK is returned instead of 404 when the
+    /// application specified in the request does not exist.
+    ignore_missing_app: Option<bool>,
```

Comment:
Sorry to bikeshed, but this is public API. Do we have a better name in mind?

Reply:
As usual, I went with the most in-your-face thing I could think of. I think it's fairly clear, if perhaps a bit long. If you want something shorter maybe `stateless=true`? I'm afraid that's a bit too opaque/vague though.

Reply:
FWIW `ignore_missing_app` seems pretty intuitive to me as well.

Reply:
It just feels a bit JAVA to me "IgnoreMissingAppAndReturn404IfNotExist" <-- I wonder if there's a more generic and short name for this.

Reply:
A day later, I still think `ignore_missing_app` is good, I can't think of better alternatives.

Reply:
Still not a fan. "missing" doesn't really mean what we want, and "ignore" also doesn't fully capture what we are after.

* Maybe `skip_missing_app` feels like it's maybe better, but still feels lacking.
* `skip_no_app`? too vague/broken English
* `skip_nonexistent_app`? too long
* `skip_absent_app`? blah.

Maybe `ignore_missing_app` is the best we have. Any other thoughts? If no, I guess we can go ahead. :|